src/applications/edu-benefits/1995/containers/StemEligibilityView.jsx

```diff
@@ -0,0 +1,274 @@
+      exhaustionOfBenefits || exhaustionOfBenefitsAfterPursuingTeachingCert;
+    const exhaustionOfBenefitsText =
+      'Have used all your education benefits or are within 6 months of doing so';
+    return (
+      <li className="vads-u-margin-bottom--0">
+        <span className="fa-li">
+          <i
+            className={this.iconClass(exhaustionOfBenefitsCheck)}
+            title={this.iconText(
+              exhaustionOfBenefitsCheck,
+              exhaustionOfBenefitsText,
+            )}
+            aria-hidden="true"
+          />
+        </span>
+        {exhaustionOfBenefitsText}
+      </li>
+    );
+  };
+
+  renderIsEnrolledStemCheck = () => {
+    const { isEnrolledStem, isPursuingTeachingCert } = this.props;
+
+    const isEnrolledStemCheck = isEnrolledStem || isPursuingTeachingCert;
+
+    const isEnrolledStemText =
+      'Are enrolled in a STEM undergraduate degree program, or have earned a STEM degree and are now pursuing a teaching certification';
+
+    return (
+      <li className="vads-u-margin-bottom--0">
+        <span className="fa-li">
+          <i
+            className={this.iconClass(isEnrolledStemCheck)}
+            title={this.iconText(isEnrolledStemCheck, isEnrolledStemText)}
+            aria-hidden="true"
+          />
+        </span>
+        Are enrolled in a STEM undergraduate degree program, <strong>or</strong>{' '}
+        have earned a STEM degree and are now pursuing a teaching certification
+      </li>
+    );
+  };
+
+  renderChecks = () => (
+    <div>
+      <p className="vads-u-margin-bottom--1">
+        <span className="vads-u-font-family--serif heading-level-4">
+          Based on you responses, it appears you're not eligible.
+        </span>
+        <br />
+        <br />
+        <b>Your responses:</b>
+      </p>
+      <ul
+        className="fa-ul vads-u-margin-left--3 vads-u-margin-top--0p5 stem-eligibility-ul"
+        role="list"
+      >
+        {this.renderIsEdithNourseRogersScholarshipCheck()}
+        {this.renderExhaustionOfBenefitsCheck()}
+        {this.renderIsEnrolledStemCheck()}
+      </ul>
+    </div>
+  );
+
+  renderErrorMessage = () => {
+    const { errors, showErrors } = this.props;
+    if (showErrors) {
+      return (
+        <div>
+          {errors.map(error => (
+            <span key={error} className="usa-input-error-message" role="alert">
+              {error}
+            </span>
+          ))}
+        </div>
+      );
+    }
+    return null;
+  };
+
+  renderDetermineEligibility = () => {
+    const { showErrors, determineEligibility } = this.props;
+    const id = 'root_determineEligibility';
+    const divClassName = classNames(
+      'form-radio-buttons',
+      showErrors ? 'usa-input-error' : '',
+    );
+    const legendClassName = classNames(
+      'schemaform-label',
+      showErrors ? 'usa-input-error-label' : '',
+    );
+    return (
+      <div className={divClassName}>
+        <legend className={legendClassName}>
```

Comment:
This radio group needs to be wrapped in a `<fieldset>` as the first child inside the `<div>`

---
![Screen Shot 2020-02-24 at 5 17 20 PM](https://user-images.githubusercontent.com/934879/75199971-18878900-572a-11ea-83df-e9155c0d6803.png)

Reply:
wrapped in fieldset